specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingSubscription.json

```diff
@@ -109,535 +241,1226 @@
+          "description": "The name of the customer.",
+          "type": "string",
+          "readOnly": true
+        },
+        "customerName": {
+          "description": "The ID that uniquely identifies a customer.",
+          "type": "string",
+          "readOnly": true
+        },
         "displayName": {
-          "description": "The name of the subscription.",
-          "readOnly": true,
+          "description": "The name of the billing subscription.",
           "type": "string"
         },
-        "invoiceSectionDisplayName": {
-          "description": "The name of the invoice section to which the subscription is billed.",
-          "readOnly": true,
-          "type": "string"
+        "enrollmentAccountId": {
+          "description": "The enrollment Account ID associated with the subscription. This field is available only for the Enterprise Agreement Type billing accounts.",
+          "type": "string",
+          "readOnly": true
+        },
+        "enrollmentAccountDisplayName": {
+          "description": "The enrollment Account name associated with the subscription. This field is available only for the Enterprise Agreement Type billing accounts.",
+          "type": "string",
+          "readOnly": true
+        },
+        "enrollmentAccountSubscriptionDetails": {
+          "description": "Enrollment Account Subscription details. This field is available only for the Enterprise Agreement Type billing accounts.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/EnrollmentAccountSubscriptionDetails"
+            }
+          ]
         },
         "invoiceSectionId": {
-          "description": "The ID of the invoice section to which the subscription is billed.",
+          "description": "The fully qualified ID that uniquely identifies an invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "invoiceSectionDisplayName": {
+          "description": "The name of the invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "invoiceSectionName": {
+          "description": "The ID that uniquely identifies an invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "nextBillingCycleDetails": {
+          "description": "Next billing cycle details of the subscription.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/NextBillingCycleDetails"
+            }
+          ]
+        },
+        "offerId": {
+          "description": "The offer ID for the subscription. This field is only available for the Microsoft Online Services Program billing accounts.",
+          "type": "string",
+          "readOnly": true
+        },
+        "productCategory": {
+          "description": "Category of the subscription. Possible values include: ReservationOrder, SavingsPlanOrder, AzureSupport, UsageBased, Hardware, SaaS, SeatBased, and Software.",
```

Comment:
We should remove the possible values from the description as they are defined as an enum.